backend/validators/task.js

```diff
@@ -0,0 +1,11 @@
+const { body, param } = require('express-validator');
+
+const store = [body('name').trim().notEmpty().withMessage('Task name is required')];
+const update = [
+    ...store,
+    ...destroy,
```

Comment:
destroy needs to be above update if you want to do this

Reply:
done